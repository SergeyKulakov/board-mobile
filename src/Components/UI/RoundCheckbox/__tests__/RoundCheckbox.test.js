import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { RoundCheckbox as Component } from '../index'

describe('RoundCheckbox', () => {
  const defaultProps = {
    isActive: true,
    disabled: false,
    onClick: () => null,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
