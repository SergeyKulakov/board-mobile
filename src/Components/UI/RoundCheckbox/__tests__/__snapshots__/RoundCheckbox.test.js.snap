// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoundCheckbox render snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  disabled={false}
  hitSlop={
    Object {
      "bottom": 15,
      "left": 15,
      "right": 15,
      "top": 15,
    }
  }
  onPress={[Function]}
>
  <Styled(View)>
    <Styled(View)
      isActive={true}
    />
  </Styled(View)>
</TouchableOpacity>
`;
